vignettes/adrg.qmd

```diff
@@ -0,0 +1,785 @@
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+|                                              |                                |          |        |                         |           |                                                                               |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+| ADLBC - Analysis Dataset Lab Blood Chemistry | BASIC DATA STRUCTURE           |          | x      |                         |           | One or more records per subject per analysis parameter per analysis timepoint |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+|                                              |                                |          |        |                         |           |                                                                               |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+| ADTTE - AE Time To 1st Derm. Event Analysis  | BASIC DATA STRUCTURE           | x        | x      |                         |           | One or more records per subject per analysis parameter per analysis timepoint |
++----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
+
+: {tbl-colwidths="\[20,20,10,10,20,10,10\]"}
+
+### ADSL - Subject-Level Analysis Dataset
+
+The subject level analysis dataset (ADSL) contains required variables for demographics, treatment groups, and population flags. In addition, it contains other baseline characteristics that were used in both safety and efficacy analyses. All patients in DM were included in ADSL. The following are the key population flags are used in analyses for patients: • SAFFL – Safety Population Flag (all patients having received any study treatment) • ITTFL – Intent-to-Treat Population Flag (all randomized patients)
+
+### ADADAS - ADAS-COG Analysis Dataset
+
+ADADAS contains analysis data from the ADAS-Cog questionnaire, one of the primary efficacy endpoints. It contains one record per subject per parameter (ADAS-Cog questionnaire item) per VISIT. Visits are placed into analysis visits (represented by AVISIT and AVISITN) based on the date of the visit and the visit windows.
```

Comment:
Did we want to talk about the issue with slotting here?  I know we reverted back to original, but was thinking pointing this out might be handy?

Reply:
I understand where you're coming from, but I think that this issue will be addressed outside of the Pilot 3 submission through a blog by Joe and Ning, per the recent R Consortium RSWG minutes from the time stamp of this comment.  Though we point to the more bigger issue as captured in the QC Findings. In regards to slotting/windowing, maybe we can add a sentence to mention `... visit windows. More information on the analysis visit windows can be found in the Statistical Analysis Plan`. What do you think @kaz462 ?